metaconfig-core/shared/src/main/scala/metaconfig/Input.scala

```diff
@@ -81,17 +81,21 @@ object Input {
     override def toString = s"""Input.VirtualFile("$path", "...")"""
   }
 
-  final case class File(file: Path, charset: Charset)
+  final case class File(file: Path, charset: Charset, contents: Predef.String)
```

Comment:
I'm afraid this is a binary breaking change meaning existing libraries using of `Input.File` would crash at runtime. It's very difficult to evolve case classes in a binary compatible way, it's usually better to create new classes instead.